java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/map/prunability/ScanByPrunedCompositeKeyBenchmarkRandomAccess.java

```diff
@@ -15,102 +15,14 @@
  */
 package com.hazelcast.simulator.tests.map.prunability;
 
-import com.hazelcast.config.MapConfig;
-import com.hazelcast.config.PartitioningAttributeConfig;
-import com.hazelcast.map.IMap;
-import com.hazelcast.simulator.hz.HazelcastTest;
-import com.hazelcast.simulator.hz.IdentifiedDataSerializablePojo;
-import com.hazelcast.simulator.test.annotations.Prepare;
-import com.hazelcast.simulator.test.annotations.Setup;
-import com.hazelcast.simulator.test.annotations.Teardown;
-import com.hazelcast.simulator.test.annotations.TimeStep;
-import com.hazelcast.simulator.worker.loadsupport.Streamer;
-import com.hazelcast.simulator.worker.loadsupport.StreamerFactory;
-import com.hazelcast.sql.SqlResult;
-import com.hazelcast.sql.SqlRow;
-import com.hazelcast.sql.SqlService;
+import java.util.concurrent.ThreadLocalRandom;
 
-import java.io.Serializable;
-import java.util.List;
-import java.util.Random;
+public class ScanByPrunedCompositeKeyBenchmarkRandomAccess extends ScanByPrunedCompositeKeyBenchmarkBase {
+    private final ThreadLocalRandom random = ThreadLocalRandom.current();
```

Comment:
are you sure that given instance is created and invoked exclusively on a single thread? caching `ThreadLocalRandom.current()` does not give much benefit and complicates a lot.

Reply:
Fixed.